src/test/java/gov/cms/madiejavamodels/MeasurePopulationTest.java

```diff
@@ -0,0 +1,33 @@
+package gov.cms.madiejavamodels;
+
+import gov.cms.madiejavamodels.measure.MeasurePopulation;
+import org.junit.Test;
+
+
+public class MeasurePopulationTest {
+  @Test
+  public void testPopulationToCode() {
+    assert MeasurePopulation.INITIAL_POPULATION.toCode() == "initial-population";
```

Comment:
For Strings in Java, recommend using the `equals()` method over the `==` operator. The earlier will compare content, while the latter will performance a reference comparison (same memory address).

Reply:
good point! updated now. Thanks!